api/src/services/stardust/influx/influxDbClient.ts

```diff
@@ -115,9 +115,11 @@ export abstract class InfluxDbClient {
      * @returns Boolean representing that the client ping succeeded.
      */
     public async buildClient(): Promise<boolean> {
-        const protocol = "https";
+        const protocol = this._network.analyticsInfluxDbProtocol || 'https';
         const network = this._network.network;
-        const host = this._network.analyticsInfluxDbEndpoint;
+        var [host, portString] = this._network.analyticsInfluxDbEndpoint.split(':');
+        // Parse port string to int, or use default port for protocol
+        const port = parseInt(portString) || (protocol === 'https' ? 443 : 80);
```

Comment:
This computed `port` is not actually used below. There it's still hardcode 443. Other than that LGTM.